usaspending_api/bulk_download/v2/views.py

```diff
@@ -160,10 +173,40 @@ def post(self, request):
 
         # create download job in database to track progress. Starts at 'ready'
         # status by default.
-        download_job = BulkDownloadJob(job_status_id=JOB_STATUS_DICT['ready'],
-                                       file_name=timestamped_file_name)
+        download_job_kwargs = {'job_status_id': JOB_STATUS_DICT['ready'],
+                               'json_request': json.dumps(order_nested_object(json_request)),
+                               'file_name': timestamped_file_name}
+        award_levels = json_request['award_levels']
+        award_types = json_request['filters']['award_types']
```

Comment:
I saw the check for required parameters on lines 152-154. However I didn't see anything for 'filters' dict keys.

Reply:
That gets checked via 
`csv_sources = self.get_csv_sources(json_request)` on line 168 which calls 
`queryset = self.process_filters(json_request['filters'], award_level)` on line 451 which calls
```
for required_param in ['award_types', 'agency', 'date_type', 'date_range']:
            if required_param not in filters:
                raise InvalidParameterException('{} filter not provided'.format(required_param))
``` on lines 377-379

Reply:
thanks